tutorials/international/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: tutorial
+title: "International Flutter Apps"
```

Comment:
Suggested: Internationalizing Flutter Apps (more action oriented)

Reply:
I did start with that. IMHO it's too many syllables :-)

Reply:
The title is for SEO, so "Internationalizing" or "Internationalization" is likely what people will search for.